ShutdownMenuWithIcons@LLOBERA/files/ShutdownMenuWithIcons@LLOBERA/applet.js

```diff
@@ -1,23 +1,23 @@
 // Shutdown Menu With Icons Cinnamon Applet
-// Developed by Nicolas LLOBERA - nllobera@gmail.com from the System Shutdown and Restart Applet by Shelley
-// version: 2.2 (28-08-2013)
+// Developed by Nicolas LLOBERA from the System Shutdown and Restart Applet by Shelley
+// version: 2.3 (08-02-2024)
 // License: GPLv3
-// Copyright © 2013 Nicolas LLOBERA
+// Copyright © 2024 Nicolas LLOBERA
 
-const Applet = imports.ui.applet;
 const Gettext = imports.gettext;
-const UUID = "ShutdownMenuWithIcons@LLOBERA";
-const Gtk = imports.gi.Gtk;
 const Gio = imports.gi.Gio;
-const Main = imports.ui.main;
-const PopupMenu = imports.ui.popupMenu;
-const Settings = imports.ui.settings;
+const GLib = imports.gi.GLib;
+const Gtk = imports.gi.Gtk;
 const St = imports.gi.St;
 const Util = imports.misc.util;
-const GLib = imports.gi.GLib;
+const Applet = imports.ui.applet;
+const PopupMenu = imports.ui.popupMenu;
+const Settings = imports.ui.settings;
+
+const UUID = "ShutdownMenuWithIcons@LLOBERA";
 const AppletUUID = "ShutdownMenuWithIcons@LLOBERA";
```

Comment:
Just a note here, you're defining the same thing twice. You could use one and just replace every reference.